packages/core/src/CBOR/TxCBOR.ts

```diff
@@ -0,0 +1,27 @@
+// To be hoisted to a new package.
```

Comment:
Maybe just make a ticket for this instead

Reply:
ADP-2687
Will remove inline comment in a fixup commit

Reply:
2fcc3f6806e0638ede2095d4d180e262d9160f43